src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -384,6 +384,9 @@
   <data name="SortNotSupported" xml:space="preserve">
     <value>The _sort parameter is not supported.</value>
   </data>
+  <data name="TruncatedIncludes" xml:space="preserve">
+    <value>Truncated. Too many included resources</value>
```

Comment:
I think this could be phrased like: The bundle only contains the first {0} included resources, not the complete set.